log: add a barebones log config in case configuration files not present

maugustosilva opened the pull request:
Fixes #1116

aplanas commented on keylime/keylime_logging.py:
```diff
@@ -5,7 +5,20 @@
+        },
+    )
+
+    logging_config.dictConfig(logging_barebones_config)
```
Maybe a more direct approach can be:

```python
...
except KeyError:
    logging.basicConfig(format="%(asctime)s %(name)-12s %(levelname)-8s %(message)s", level=logging.DEBUG)
```

maugustosilva replied:
Nice, you're right, it is more direct (and thus better).

maugustosilva commented:
> Looks like a good plan to fallback on debug logging if no config could be found, but maybe also output a warning that we are doing that.
> 
> Can you also address the failing pylint check:
> 
> ```
> ************* Module keylime.keylime_logging
> keylime/keylime_logging.py:11:47: R1735: Consider using '{"version": 1, "formatters": {'f': {'format': '%(asctime)s %(name)-12s %(levelname)-8s %(message)s'}}, ... }' instead of a call to 'dict'. (use-dict-literal)
> ```

Fixed it and, and added the appropriate warning.
